src/Microsoft.Health.Fhir.SqlServer/Features/Schema/Migrations/13.sql

```diff
@@ -1,5 +1,11 @@
 -- Style guide: please see: https://github.com/ktaranov/sqlserver-kit/blob/master/SQL%20Server%20Name%20Convention%20and%20T-SQL%20Programming%20Style.md
 
+-- The wrapping up of the initialization script under a transaction is removed from the schema-manager tool, so adding transaction in the script itself.
+
+SET XACT_ABORT ON
```

Comment:
Do we need to do this retroactivily to all scripts?

Would it be bad if someone with new schema manager tool decide to update their script from let's say 8.sql to 9.sql?

Reply:
No, we don't need to do this for older scripts. Since when we initialize the schema we directly execute the x.sql where x is the latest.
I am not sure if I got the second part - but if it is about migration i.e. going from schema version 8 to 9 then this change won't impact because for the schema migration from we consider x.diff.sql.